connectors/service_cli.py

```diff
@@ -226,16 +146,16 @@ def run(args):
         print("Bye")  # noqa: T201
         return 0
```

Comment:
Do we still need to return?

Reply:
For now yes, otherwise we run into `if "config" in action:` down below. I'll plan to address the flow of execution in a separate PR as some things don't make too much sense IMO.